server/service/src/sync/translations/item_variant.rs

```diff
@@ -0,0 +1,97 @@
+use repository::item_variant::item_variant_row::{ItemVariantRow, ItemVariantRowRepository};
+use repository::{ChangelogRow, ChangelogTableName, StorageConnection, SyncBufferRow};
+
+use crate::sync::translations::item::ItemTranslation;
+
+use super::{
+    PullTranslateResult, PushTranslateResult, SyncTranslation, ToSyncRecordTranslationType,
+};
+
+// Needs to be added to  ()
```

Comment:
```suggestion
// Needs to be added to all_translators()
```
Are we just copying this comment around everywhere so that we know what to do next time someone copy/pastes?

Reply:
Yes, that's my understanding...

Reply:
I always find it useful